src/main/java/roomescape/member/MemberController.java

```diff
@@ -10,13 +10,18 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import java.net.URI;
+import roomescape.auth.AuthService;
+import roomescape.token.TokenRequest;
+import roomescape.token.TokenResponse;
 
 @RestController
 public class MemberController {
     private MemberService memberService;
+    private AuthService authService;
```

Comment:
위에서 알려드린 거 처럼 `final`로 선언하시는 것이 좋아보여요!
```suggestion
    private final MemberService memberService;
    private final AuthService authService;
```

Reply:
수정 진행 했어요!